config/routes.rb

```diff
@@ -887,6 +894,112 @@
                save_post
     },
 
+    :ems_middleware            => {
+      :get  => %w(
+        download_data
+        edit
+        index
+        new
+        show
+        show_list
+        tagging_edit
+        tag_edit_form_field_changed
+      ) +
+               compare_get,
+      :post => %w(
+        button
+        create
+        dynamic_checkbox_refresh
+        form_field_changed
+        listnav_search_selected
+        panel_control
+        quick_search
+        sections_field_changed
+        show
+        show_list
+        tl_chooser
+        update
+        wait_for_task
+        tagging_edit
+        tag_edit_form_field_changed
+      ) +
+               adv_search_post +
+               compare_post +
+               exp_post +
+               save_post
+    },
+
+    :middleware_server            => {
+       :get  => %w(
```

Comment:
Still not aligned...should be two space indent (so the `:` should be aligned with the `i` on previous line)